table/server/master/src/main/java/alluxio/master/table/TableMaster.java

```diff
@@ -138,4 +139,12 @@ boolean detachDatabase(String dbName)
    * @return true if the database was changed as a result
    */
   boolean syncDatabase(String dbName) throws IOException;
+
+  /**
+   * Gets a database object.
+   *
+   * @param dbName the database name
+   * @return a database object
+   */
+  Database getDatabase(String dbName) throws IOException;
```

Comment:
i don't care that much but for the most part methods are in alphabetical order in this class. Should we try to keep it that way? the transform methods are obviously not in the right order, but the rest seem to be